assignments/closure.js

```diff
@@ -1,20 +1,55 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+myClosureParent();
```

Comment:
I love that you named things literally. It truly helps show that you understand what goes where and why. Great job on this.